packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -197,9 +205,9 @@ export interface IContextualMenuItem {
   href?: string;
 
   /**
-   * A collection of submenu items
+   * A collection of submenu items, or an IContextualMenuProps object to apply for the sub menu
    */
-  items?: IContextualMenuItem[];
+  items?: IContextualMenuItem[] | IContextualMenuProps;
 
```

Comment:
We have a deprecation policy to not remove props but instead to deprecate which generates are warning and documents it. (The comment has @deprecated for documentation, and the constructor takes a deprecation map that generates a warning if the prop is used.)

So we'd update everything to use subMenuProps in examples, but if item is provided, still support that for backwards compat until major bump.